src/main/java/Orchid.java

```diff
@@ -0,0 +1,41 @@
+public class Orchid {
+    String name;
+
+    int cutOrchid;
+
+    double species;
+
+    Orchid(){
+
+    }
+
+    public Orchid(String name, int cutOrchid, double species) {
```

Comment:
Second constructor

Reply:
please check second file in day6 branch

Reply:
E.g. you need to be able to create object of class with just name provided, i.e. the public Orchid(String name){} constructor should be added, and in similar way for all the other combinations of name, cut and species